app/presenters/hub/dashboard/dashboard_presenter.rb

```diff
@@ -29,7 +29,7 @@ def selected_orgs_and_sites
               if model.instance_of? Site
                 model = available_by_id[model.parent_organization_id]
               end
-              model.coalition_id == selected_model.id
+              model&.coalition_id == selected_model.id
```

Comment:
unrelated to this ticket but noticed this will cause this page to crash sometimes; model is nil when can't find model on available_by_id[model.parent_organization_id]